docs/get-started/get-an-allocation.md

```diff
@@ -58,14 +58,18 @@ allocations, and provide information such as publications and grant data. PIs ca
 maintain all their research information under one project or if they require they
 can separate the work into multiple projects.
 
-![Request A New Allocation](images/coldfront-request-new-allocation.png)
+## How to request a new Resource Allocation
+
+### Request A New OpenStack Resource Allocation for OpenStack Project
+
+![Request A New OpenStack Resource Allocation](images/coldfront-request-new-openstack-allocation.png)
 
 !!! warning "Very Important"
     Make sure to select **NERC (OpenStack)** on Resource option and specify
     your expected Units of computing. Be mindful, you can extend your current resource
     allocations on your current project later on.
 
-## Resource Allocation Quotas
+#### Resource Allocation Quotas for OpenStack Project
```

Comment:
was this intended to be 4 `#`'s

Reply:
This is intended to make sub section for "How to request a new Resource Allocation"

Reply:
wouldn't that be 3 #, not 4?

Reply:
It need to have 4 # as a child list for Request A New OpenStack Resource Allocation for OpenStack Project (which has 3 #)

Reply:
sorry, I was mixing up red and green, this totally should be 4.